openshift-client/script.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Why not use array params and avoid this script?

Reply:
From the discussion, this is to ensure backward compatibility. So alright with me.